src/segment-loader.js

```diff
@@ -450,6 +490,12 @@ export default class SegmentLoader extends videojs.EventTarget {
       }
     });
 
+    this.on('trackinfo', () => {
```

Comment:
Since `startingMedia_` is reset on media change now, we have to check if we have enough to append after `trackinfo`, which sets `startingMedia_`